nucleus/core/logging/src/main/java/fish/payara/enterprise/server/logging/JSONLogFormatter.java

```diff
@@ -379,6 +378,30 @@ private String jsonLogFormat(LogRecord record) {
                         .append(eventObject.toString()), level);
             }
 
+            Object[] parameters = record.getParameters();
+            if (parameters != null) {
+                for (Object parameter : parameters) {
+                    if (parameter instanceof Map) {
```

Comment:
Why not support all parameters as JsonArray and have the Map be on JsonObject within it?